Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.14.3-alpine
```

Comment:
this is great. look up how to use a builder image, so you can copy your binary into an even smaller image.